packages/sdks/javascript/README.md

```diff
@@ -74,25 +74,59 @@ getUserProfile();
 This package includes TypeScript declarations for every method.
 TypeScript versions `>= 3.8` are supported.
 
-Some methods in `Devopness SDK JavaScript` accept and return objects from the Devopness API. The type declarations for these objects will always track the latest version of the API. Therefore, if you'e using the latest version of this package, you can rely on the Devopness API documentation for checking the input and return types of each API endpoint.
+**Some methods in `Devopness SDK JavaScript` accept and return objects from the Devopness API. The type declarations for these objects will always track the latest version of the API. Therefore, if you'e using the latest version of this package, you can rely on the Devopness API documentation for checking the input and return types of each API endpoint.**
 
 ## Building and testing
 To build and test the SDK locally, follow these steps:
-1. Clone de repository
+
+## 1. Clone de repository
 ```
 git clone https://github.com/devopness/devopness.git
 ```
-2. Install missing dependencies
 
-This command will install all modules listed as dependencies in [package.json](package.json). A working Java Runtime Environment is also required. Please, check out the installation instructions for your operating system.
+## 2. Setup
+### With Docker + Makefile
+
+#### 1. Build
+```
+make build
+```
+
+#### 2. Run Tests
+```
+make test
+```
+
+### With Docker Only
```

Comment:
We should document only two ways to get things done: with docker (recommended and using the `make` commands) and without it.

I can't see any advantage of documenting a third/hybrid approach using Docker and not taking advantage of the `make` commands.
`make` is present in almost all operating systems with minimal to zero dependencies.

Let's just encourage devs to have a simpler and straightforward way to get things done without installing many dependencies on their host systems.

Reply:
resolved